fedimint-client/src/sm/executor.rs

```diff
@@ -932,7 +1039,9 @@ impl ::fedimint_core::db::DatabaseLookup for InactiveStateKeyPrefix {
 enum ActiveOrInactiveState {
     Active {
         dyn_state: DynState,
-        active_state: ActiveState,
+        // TODO: remove?
```

Comment:
I think it was used to log something, and that's about it. Remove?

Reply:
Having some timing info is good, it's currently our only way of ordering the state machine log in the DB :see_no_evil: I know that's not bullet-proof.